### DISCUSSION TITLE
OccurreceConstraint .None()

### DISCUSSION THREAD

**Discussion by ITaluone** [fluentassertions/fluentassertions#1877] [Ideas]
Hi

Is it intentional that the occurrence constraint for zero times is XXX.Times(0) and not XXX.None().

If this is a useful extension I would add that

Edit: ok.. `Contain("a", Exactly.None())` or `Contain("a", Exactly.Times(0))` is the same as `NotContain()`... would be an unnecessary adoption

**Comment by jnyrup** [comment]
You answered that perfectly yourself 😄
